specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/examples/UpdateKeyRotationPolicy-example.json

```diff
@@ -0,0 +1,61 @@
+{
+  "parameters": {
+    "vaultBaseUrl": "https://myvault.vault.azure.net/",
+    "key-name": "updateKey01",
+    "api-version": "7.3-preview",
+    "keyRotationPolicy": {
+      "lifetimeActions": [
+        {
+          "trigger": {
+            "timeAfterCreation": "P90D"
+          },
+          "action": {
+            "type": "Rotate"
+          }
+        },
+        {
+          "trigger": {
+            "timeBeforeExpiry": "P30D"
+          },
+          "action": {
+            "type": "Notify"
+          }
+        }
+      ],
+      "attributes": {
+        "rotationEnabled": true,
+        "rotationTime": "P2Y"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "https://myvault.vault.azure.net/keys/updateKey01/rotationpolicy",
+        "lifetimeActions": [
+          {
+            "trigger": {
+              "timeAfterCreation": "P90D"
+            },
+            "action": {
+              "type": "Rotate"
+            }
+          },
+          {
+            "trigger": {
+              "timeBeforeExpiry": "P30D"
+            },
+            "action": {
+              "type": "Notify"
+            }
+          }
+        ],
+        "attributes": {
+          "expirationTime": "2Y",
+          "created": 1482188947,
+          "updated": 1482188948
+        }
+      }
```

Comment:
The `expirationTime` here should be `P2Y`, not `2Y`.

Reply:
Fixed it